docs/feature-gates.md

```diff
@@ -209,3 +209,45 @@ Refer to this [document](egress.md) for more information.
 This feature is currently only supported for Nodes running Linux and "encap"
 mode. The support for Windows and other traffic modes will be added in the
 future.
+
+### AntreaIPAM
```

Comment:
Many files are not in this PR's scope and were already modified by previous PRs.
Is there anything wrong in this PR's rebase progress?

Reply:
I'm not sure why this happens when base branch is updated - I rebased again and force pushed.